core/templates/domain/admin/admin-backend-api.service.spec.ts

```diff
@@ -198,3 +199,116 @@ describe('Admin backend api service', () => {
     })
   );
 });
+
+describe('Admin Backend API service for Jobs Tab', () => {
```

Comment:
I thought that it would be appropriate to distinguish tests as we are fitting different tabs(misc, jobs, roles) in the same admin-backend-api service as you suggested before.